packages/contracts/test/extensions/compliant_forwarder.ts

```diff
@@ -0,0 +1,318 @@
+        let erc20TokenB: DummyERC20TokenContract;
+        [erc20TokenA, erc20TokenB, zrxToken] = await erc20Wrapper.deployDummyTokensAsync(
+            numDummyErc20ToDeploy,
+            constants.DUMMY_TOKEN_DECIMALS,
+        );
+        defaultMakerAssetAddress = erc20TokenA.address;
+        defaultTakerAssetAddress = erc20TokenB.address;
+        zrxAssetData = assetDataUtils.encodeERC20AssetData(zrxToken.address);
+        // Deploy Yes Token
+        const yesTokenInstance = await YesComplianceTokenContract.deployFrom0xArtifactAsync(
+            artifacts.YesComplianceToken,
+            provider,
+            txDefaults,
+        );
+        // Create proxies
+        const erc20Proxy = await erc20Wrapper.deployProxyAsync();
+        await erc20Wrapper.setBalancesAndAllowancesAsync();
+        // Deploy Exchange congtract
+        exchangeInstance = await ExchangeContract.deployFrom0xArtifactAsync(
+            artifacts.Exchange,
+            provider,
+            txDefaults,
+            zrxAssetData,
+        );
+        exchangeWrapper = new ExchangeWrapper(exchangeInstance, provider);
+        // Register proxies
+        await exchangeWrapper.registerAssetProxyAsync(erc20Proxy.address, owner);
+        await erc20Proxy.addAuthorizedAddress.sendTransactionAsync(exchangeInstance.address, {
+            from: owner,
+        });
+        // Default order parameters
+        const defaultOrderParams = {
+            exchangeAddress: exchangeInstance.address,
+            makerAddress: compliantMakerAddress,
+            feeRecipientAddress,
+            makerAssetData: assetDataUtils.encodeERC20AssetData(defaultMakerAssetAddress),
+            takerAssetData: assetDataUtils.encodeERC20AssetData(defaultTakerAssetAddress),
+            makerAssetAmount,
+            takerAssetAmount,
+            makerFee: Web3Wrapper.toBaseUnitAmount(new BigNumber(100), DECIMALS_DEFAULT),
+            takerFee: Web3Wrapper.toBaseUnitAmount(new BigNumber(150), DECIMALS_DEFAULT),
+        };
+        const privateKey = constants.TESTRPC_PRIVATE_KEYS[accounts.indexOf(compliantMakerAddress)];
+        orderFactory = new OrderFactory(privateKey, defaultOrderParams);
+        // Deploy Compliant Forwarder
+        compliantForwarderInstance = await CompliantForwarderContract.deployFrom0xArtifactAsync(
+            artifacts.CompliantForwarder,
+            provider,
+            txDefaults,
+            exchangeInstance.address,
+            yesTokenInstance.address,
+        );
+        /*
+        const compliantForwarderContract = new CompliantForwarderContract(
+            compliantForwarderInstance.abi,
+            compliantForwarderInstance.address,
+            provider,
+        );
+        forwarderWrapper = new ForwarderWrapper(compliantForwarderContract, provider);
+        */
+        // Initialize Yes Token
+        await yesTokenInstance._upgradeable_initialize.sendTransactionAsync({ from: owner });
```

Comment:
There's a ton of extra logic here that doesn't actually affect our tests and could be initialized to null values (or just use `DummyERC721Token`).